src/Nancy/GlobalizationConfiguration.cs

```diff
@@ -15,15 +15,22 @@ public class GlobalizationConfiguration
         /// <summary>
         /// Initializes a new instance of the <see cref="GlobalizationConfiguration"/> class
         /// </summary>
-        /// <param name="supportedCultureNames"></param>
-        public GlobalizationConfiguration(IEnumerable<string> supportedCultureNames)
+        /// <param name="supportedCultureNames">An array of supported cultures</param>
+        /// <param name="defaultCulture">The default culture of the application</param>

```

Comment:
My thoughts were that his is just not for the culture conventions, but the configuration in general. If you say you support a + b and then say c should be the default, that's an invalid configuration if you ask me. This means that you can't reliably check `DefaultCulture` from anywhere without also validating it against `SupportedCultureNames`.